React/CoreModules/RCTAppearance.mm

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import "RCTAppearance.h"
+
+#import <FBReactNativeSpec/FBReactNativeSpec.h>
+#import <React/RCTEventEmitter.h>
+
+#import "CoreModulesPlugins.h"
+
+using namespace facebook::react;
+
+NSString *const RCTAppearanceColorSchemeLight = @"light";
+NSString *const RCTAppearanceColorSchemeDark = @"dark";
+
+static NSString *RCTColorSchemePreference(UITraitCollection *traitCollection)
+{
+#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
+  if (@available(iOS 13.0, *)) {
+    static NSDictionary *appearances;
+    static dispatch_once_t onceToken;
+
+    dispatch_once(&onceToken, ^{
+      appearances = @{
+                      @(UIUserInterfaceStyleLight): RCTAppearanceColorSchemeLight,
+                      @(UIUserInterfaceStyleDark): RCTAppearanceColorSchemeDark
```

Comment:
we should add back `@(UIUserInterfaceStyleUnspecified): RNCAppearanceColorSchemeNoPreference` here to better match related [web spec](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme). see [related discussion](https://github.com/react-native-community/discussions-and-proposals/issues/126#issuecomment-521070504)